centreon/www/include/home/customViews/triggers/loadServiceFromHost.php

```diff
@@ -87,7 +87,9 @@
         while ($row = $stmt->fetch(\PDO::FETCH_ASSOC)) {
             $xml->startElement('option');
             $xml->writeElement('id', $row['service_id']);
-            $xml->writeElement('label', $row['service_description']);
+            # For meta services, use display_name column instead of service_description
```

Comment:
The `#` character is unconventional for comments in 2023, except for specific Attribute case.
Please use `//`